appbidding_googleadmanager/build.gradle

```diff
@@ -21,5 +21,5 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'com.google.android.gms:play-services-ads:19.1.0'
-    implementation 'com.criteo.publisher:criteo-publisher-sdk:3.10.0'
+    implementation 'com.criteo.publisher:criteo-publisher-sdk:4.0.0'
```

Comment:
Please also update `versionName` (a few lines above) to `4.0.0.0`

Reply:
Updated